src/libs/TransactionUtils.ts

```diff
@@ -76,6 +80,19 @@ function isManualRequest(transaction: Transaction): boolean {
     return getRequestType(transaction) === CONST.IOU.REQUEST_TYPE.MANUAL;
 }
 
+function getTransactionsByActionType(actionType?: string): Array<OnyxEntry<Transaction>> {
+    return (
+        Object.values(allTransactions ?? {})
+            .filter(
+                (transaction): transaction is Transaction =>
+                    transaction != null && (actionType === CONST.IOU.TYPE.SPLIT ? isSplitRequest(transaction) : getRequestType(transaction) === actionType),
```

Comment:
This is a little bit hard to read, I think a helper method (something like `getTransactionActionType`, essentially a combination of `isSplitRequest` and `getRequestType`) would be useful here